_candidates/jeff-sheehy.md

```diff
@@ -1,39 +1,71 @@
 ---
-id: 28
+id: 47
 name: Jeff Sheehy
 photo_url: null
-website_url: "http://jeffsheehy.org/"
+website_url: null
```

Comment:
This data can be automatically pulled in by the API now by modifying Sheehy's row in the "All Candidates" tab of the candidates spreadsheet.

Reply:
(I've added the website here, so it should come back next time we sync the data.)